config/default/filter.format.basic_html_without_media.yml

```diff
@@ -17,7 +17,7 @@ filters:
     status: true
     weight: -10
     settings:
-      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id class> <h3 id class> <h4 id class> <h5 id class> <h6 id class> <table class> <caption class> <tbody class> <thead> <tfoot> <th scope class> <td class> <tr> <a id href name target hreflang data-entity-substitution data-entity-type data-entity-uuid title class> <p class> <div class> <br> <span class> <i class> <hr>'
+      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id class=""> <h3 id class=""> <h4 id class=""> <h5 id class=""> <h6 id class=""> <caption class=""> <tbody class=""> <thead> <tfoot> <th scope class=""> <td class=""> <tr> <div class=""> <br> <i class=""> <hr> <table class=" hs-table--borderless"> <a id href name target hreflang data-entity-substitution data-entity-type data-entity-uuid title class=" hs-button hs-button--big hs-secondary-button hs-more-link hs-external-link hs-file-link hs-pdf-link hs-private-link"> <p class=" hs-font-lead hs-font-splash hs-short-line-length hs-breakout-box hs-caption hs-credits"> <span class=" hs-emphasized-text">'
```

Comment:
The allowed attributes needs to be fixed.
Related to this issue: https://www.drupal.org/project/drupal/issues/2710427